diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -413,11 +417,12 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
                     getOrderedCallExpressions(psi, it)
                 }
             }
-            ?.dropWhile { !it.treeParent.textContains('(') && !it.treeParent.textContains('{') }
+            ?.takeIf { it.first() != this }
```

Comment:
We should have an option in configuration to set minimum number of calls to start putting newlines.